lms/djangoapps/certificates/apis/v0/urls.py

```diff
@@ -0,0 +1,23 @@
+""" Certificates API v0 URLs. """
+
+from django.conf import settings
+from django.conf.urls import patterns, url, include
+
+from lms.djangoapps.certificates.apis.v0 import views
+
+
+CCX_URLS = patterns(
+    '',
+    url(
+        r'^{username}/{course_id}?$'.format(
+            username=settings.USERNAME_PATTERN,
+            course_id=settings.COURSE_ID_PATTERN
+        ),
+        views.CertificatesDetailView.as_view(), name='detail'
+    ),
+)
+
+urlpatterns = patterns(
```

Comment:
`patterns()` here is deprecated: https://docs.djangoproject.com/en/1.9/releases/1.8/#django-conf-urls-patterns